packages/graphql/src/translate/where/create-where-predicate.ts

```diff
@@ -37,46 +36,25 @@ export function createCypherWherePredicate({
     context: Context;
     element: GraphElement;
 }): CypherBuilder.Predicate | undefined {
-    const mappedProperties = mapPropertiesToOperators({
-        whereInput,
-        targetElement,
-        element,
-        context,
+    const whereFields = Object.entries(whereInput);
+
+    const predicates = whereFields.map(([key, value]): CypherBuilder.Predicate | undefined => {
+        if (key === "OR") {
```

Comment:
The function `mapPropertiesToPredicates` could potentially be removed by refactoring the following: 
```suggestion
        if (key === "OR" || key === "AND") {
            const nested = value.map((v) => {
                return createWherePredicate({ whereInput: v, element, targetElement, context });
            }
            if (key === "OR") {
                return CypherBuilder.or(...nested);
            }
            return CypherBuilder.and(...nested);
    });
```

Reply:
That's a great point, I'll make this change.
In the last commit, I also further refactored this by extracting the logic inside `if (key === "OR" || key === "AND") {` to a separate function. Let me know what you think of that

Reply:
It looks good to me!